egs/wsj/s5/steps/segmentation/convert_rttm_to_utt2spk_and_segments.py

```diff
@@ -0,0 +1,79 @@
+#! /usr/bin/env python
+
+"""This script converts an RTTM with
```

Comment:
Can you please give an example here that will help to illustrate what an RTTM file looks like, and try to describe what an RTTM is, mentioning that it's a NIST format.  (I'd suggest something more specific but I am hazy on this).